src/test/java/be/syntra/devshop/DevshopFront/controllers/SearchControllerTest.java

```diff
@@ -200,11 +205,10 @@ void canSortArchivedProductsTest(String url) throws Exception {
         final ProductList dummyProductList = getDummyProductList();
         final SearchModel searchModel = SearchModel.builder().searchRequest("").searchResultView(true).selectedCategories(new ArrayList<>()).build();
         final CartDto dummyCart = getCartWithMultipleNonArchivedProducts();
```

Comment:
dummyCart is no longer used

Reply:
removed

Reply:
better check the complete class again and perhaps use ctrl-F. There are still several unused dummyCarts